src/tests/banner.test.js

```diff
@@ -64,6 +73,6 @@ describe('ProfileBanner', () => {
     )
     expect(getByText('Unfollow')).toBeInTheDocument()
     fireEvent.click(getByText('Unfollow'))
-    expect(getByText('Follow')).toBeInTheDocument()
+    expect(axios.post).toHaveBeenCalledTimes(1)
```

Comment:
hi this a comment testing to see if the mocked function got called once on click